gatsby/src/components/info.js

```diff
@@ -25,10 +25,48 @@ function URLify(string) {
 
 const Info = ({ uri, location, data, yaml, ...rest }) => {
   const { title, language } = useHelmetTags(uri, data[yaml]);
+  const [emailCollectionFormState, setEmailCollectionFormState] = React.useState({});
+  const [emailCollectionFormMessage, setEmailCollectionFormMessage] = React.useState({});
+
+  const handleEmailCollectionInputChange = (e) => {
+    setEmailCollectionFormState({ ...emailCollectionFormState, [e.target.name]: e.target.value })
+  };
+
+  const encode = (data) => {
```

Comment:
I think this can protect us from cross-site scripting as long as we don't try to use the form inputs for anything other than listing on the Netlify dashboard, but you may know more about this than I do.